.travis.yml

```diff
@@ -16,4 +16,4 @@ matrix:
     script:
       - make dev
     after_success:
-      - make travis_coverage
+      - travis_wait make travis_coverage
```

Comment:
No, PD won't run 20m+.

Sometimes `make travis_coverage` may run more than 10 minutes and outputs nothing, that may cause **travis build error**, because travis thinks it is a stall job.

Here `travis_wait` extends timeout, so that we prevent stall jobs.